client/docker/app/config/generate_parameters_yml.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
```

Comment:
Do we need to have all the parameters inlined here? Couldn't we just use envsubst again and concatenate files together which contain placeholders? Not a deal breaker, but this isn't as clear as it might be.